### DISCUSSION TITLE
auto home question

### DISCUSSION THREAD

**Discussion by Jyhade101** [Jyers/Marlin#370] [Q&A] [answered]
ok, so i finally got everything to compile(because i was dumb and didnt put the right file in the right directory)... 

Heres my issue. im running and ender 3 v2, relatively stock except im trying to run a Hero Me Gen5 setup with dual fans etc. 
Well, when i edit the firmware to set the y_Min_pos -9 and or the Manual_y_home_position -9, no matter what i do, auto home always jumps off the bed about 9mm forward. What am i doing wrong? Id really like to use this setup but im seriously to the point im about to scrap it because no matter what i change the values, im always off the bed to begin with. 

Yes im a Marlin noob, but after countless videos and reading through how to change this and that and every other thing, im about as lost as i can be. 

Id Greatly Appreciate some help and im not quite sure where else to turn. Im using the no runout no probe build which works great but id really like to figure out wth im doing wrong or if this is something within the system that cannot be changed? which, i guess i would think it should be. 
also if i go to the console and actually dial the y axis back to 0(with what ive got right now at auto home it shows -9) it is exactly where it should be..

**Comment by tiagofreire-pt** [comment]
> Well, when i edit the firmware to set the y_Min_pos -9 and or the Manual_y_home_position -9, no matter what i do, auto home always jumps off the bed about 9mm forward. What am i doing wrong? Id really like to use this setup but im seriously to the point im about to scrap it because no matter what i change the values, im always off the bed to begin with.

The X axis is left to right. The Y axis is front to back. You're changing the Y axis, that's why it jumps the bed forward.

**Reply by Jyhade101** [reply]
yes because the mount sets the hotend 9mm further forward

**Reply by Jyhade101** [reply]
trying to compensate for that so that my hotend is back on the bed

**Comment by Jyers** [comment, marked as answer]
Is the issue that the autohome is going off the bed? If so this is not an issue. Autohome needs to go all the way to the endstop, even if that is off the bed. However, if you set your y min position to -9, it will go off the bed when homing, but it will consider that point -9. Therefore since your slicer is most likely set to see the bed as 0 - 335, the printer will never go off the bed, and your prints will be centered.

**Comment by Jyhade101** [comment]
Yes it goes off the bed but as I read over the weekend, and you stated,
this happens intentionally to read the end stops. The -9 on the y axis
plants the nozzle almost exactly where the nozzle was before i set up the
new cooling system.

Before i was under the impression that when that was set, it was supposed
to put the home position back on the bed
